scripts/bench

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+set -euo pipefail
+
+if [ $# -ne 2 ]; then
+  cat <<EOF
```

Comment:
Done, though it had to be `cat >2 <<EOF ..... EOF\n >2` or nothing would come out.